### DISCUSSION TITLE
Which `phoneNumber` information shell be used for KYC-match [southbound-call]

### DISCUSSION THREAD

**Discussion by maxl2287** [camaraproject/KnowYourCustomer#146] [Q&A] [answered]
Just a short question based on v0.1.1 for a possible implementation:

In the request of KYC-Match the `phoneNumber` is inside of the request.
When I want to call against a southbound API to get some user-information based on this `phoneNumber`, which `phoneNumber` information shell I use?

The one from the request or based on the accessToken.

I prefer some information from the accessToken from a data-privacy perspective.

Wdyt?

Thanks in advance. :)

**Comment by HuubAppelboom** [comment, marked as answer]
@maxl2287  Currently you would nee to provide an error in case there is a mismatch,

Hi, there is a discussion going on about this in commonalities, please take a look at 
https://github.com/camaraproject/Commonalities/issues/259
